src/modules/Scraper/README.md

```diff
@@ -9,14 +9,14 @@ It is using a well known extension called `I don't care about cookies` to preven
 
 In order to have the latest code of the extension, you need to manually download it from the chrome extensions website https://chrome.google.com/webstore/detail/i-dont-care-about-cookies/fihnjjcciajhdojfnbdddfaoknhalnja?hl=en and install it in Brave browser.
 
-**NOTE** if you're using chrome, you will need to manually change the value of BROWSER_EXTENSIONS_FOLDER in the `copy-to-folder.sh` file
+**NOTE** if you're using chrome (instead of Brave), you will need to manually change the value of BROWSER_EXTENSIONS_FOLDER in the `copy-to-folder.sh` file
```

Comment:
You should not presuppose the type of browser.

I suggest :

---

**Configuration**

You need to configure the `BROWSER_EXTENSIONS_FOLDER` value in the `copy-to-folder.sh` file. By default this uses the standard Brave Browser installation path on macOS: `~/Library/Application\ Support/BraveSoftware/Brave-Browser/Default/Extensions/$EXTENSION_ID`

---

Reply:
DONE